scripts/exgfs_atmos_post.sh

```diff
@@ -314,11 +311,10 @@ else   ## not_anl if_stime
             fi
           fi
```

Comment:
These lines now have a smaller indentation than the outer blocks.

Reply:
Fixed.